scipy/sparse/linalg/_expm_multiply.py

```diff
@@ -91,7 +91,7 @@ def traceest(A, m3, seed=None):
 
 def _ident_like(A):
     # A compatibility function which should eventually disappear.
-    if scipy.sparse.isspmatrix(A):
+    if scipy.sparse.issparse(A):
         return scipy.sparse._construct.eye(A.shape[0], A.shape[1],
```

Comment:
If given a sparse array, this will return a spmatrix. We should wrap this such that it returns a sparse array when `A` is a sparse array.